website/src/utils/fetch.js

```diff
@@ -9,7 +9,7 @@ export function getDataServerAddress() {
     if (process.env.VUE_APP_DEPLOY_MODE === "local") {
         return `//${window.location.hostname}:8901`;
     }
-    return "https://data.slmn.gg";
+    return process.env.SLMNGG_DATA_SERVER;
```

Comment:
```suggestion
    return process.env.SLMNGG_DATA_SERVER || `https://data.slmn.gg`;
```
Fallback to main data server otherwise it will try and use whatever page it's on (eg bpl.slmn.gg) and it won't work